pychebfun/chebfun.py

```diff
@@ -318,7 +321,7 @@ def chebfun(f=None, domain=[-1,1], N=None, chebcoeff=None,):
     """
     Create a Chebyshev polynomial approximation of the function $f$ on the interval :math:`[-1, 1]`.
 
-    :param callable f: Python, Numpy, or Sage function
+    :param callable f: Python, Numpy, Sage or wrapped C function
```

Comment:
Please change to: `:param callable f: any numerical function` or something of that kind.